project/en-US/compensations/compensations.xml

```diff
@@ -1,72 +1,265 @@
+        interface to get handlers restored from the object store.
+      </para>
+      <para>
+        You can find more valuable information in the wiki article
+        <ulink url="https://developer.jboss.org/wiki/XTSRecoveryInternals">XTS Recovery Internals</ulink>.
+      </para>
+    </section>
+    <section>
+      <title>Limitation</title>
+      <para>
+        Currently there is one limitation which is needed to be emphasized. If you are used to work with XTS AT transactions
+        you probably take advantage of transaction bridge functionality - ability of smoothly join XTS transaction with app server
+        container transaction under one transparent global transaction. This is not possible for compensating transaction.
+        The lack of the bridge functionality ofr the compensating transaction is addressed under jira
+        <ulink url="https://issues.jboss.org/browse/JBTM-1099">https://issues.jboss.org/browse/JBTM-1099</ulink>.
+      </para>
     </section>
+  </section>
+  <section>
+    <title>Resources</title>
+    <para>
+      This is our blog series which is valuable to check out
+    </para>
+    <itemizedlist>
+      <listitem>
+        <para>
+          <ulink url="http://jbossts.blogspot.com/2013/05/compensating-transactions-when-acid-is.html">
+            Compensating Transactions: When ACID is too much (Part 1: Introduction)
+          </ulink>
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          <ulink url="http://jbossts.blogspot.com/2013/05/compensating-transactions-when-acid-is_29.html">
+            Compensating Transactions: When ACID is too much (Part 2: Non-Transactional Resources)
+          </ulink>
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          <ulink url="http://jbossts.blogspot.com/2013/06/compensating-transactions-when-acid-is_26.html">
+            Compensating Transactions: When ACID is too much (Part 3: Cross-Domain Distributed Transactions)
+          </ulink>
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          <ulink url="http://jbossts.blogspot.com/2013/07/compensating-transactions-when-acid-is.html">
+            Compensating Transactions: When ACID is too much (Part 4: Long Lived Transactions)
+          </ulink>
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          <ulink url="http://jbossts.blogspot.com/2014/05/bringing-transactional-guarantees-to.html">
+            Bringing Transactional Guarantees to MongoDB: Part 1
+          </ulink>
+        </para>
+      </listitem>
+    </itemizedlist>
+  </section>
+  <section>
+    <title>Outstanding Important Issues</title>
+    <para>
+      Open issues are tracked in our
+      <ulink url="https://issues.jboss.org/issues/?jql=project%20%3D%20JBTM%20AND%20status%20in%20(Open%2C%20%22Coding%20In%20Progress%22%2C%20Reopened%2C%20%22Pull%20Request%20Sent%22)%20AND%20component%20%3D%20Compensations">
```

Comment:
I am not sure this is worth mentioning here as it is general that people would check out JBTM rather than specific to compensations but will approve and allow you to decide.